src/pybind/mgr/snap_schedule/fs/schedule.py

```diff
@@ -233,6 +233,14 @@ def list_schedules(cls, path, db, fs, recursive):
                                (f'{path}',))
             return [cls._from_db_row(row, fs) for row in c.fetchall()]
 
+    @classmethod
+    def list_all_schedules(cls,
+                           db: sqlite3.Connection,
+                           fs: str) -> List['Schedule']:
+        with db:
+            c = db.execute(cls.PROTO_GET_SCHEDULES + " path LIKE '%'")
+            return [cls._from_db_row(row, fs) for row in c.fetchall()]
+
```

Comment:
commit: **mgr/snap_schedule: restart old schedules**
the `List['Schedule']` type hint requires the `List` type imported from `typing`
please add that to the `import` section

Reply:
done